tox.ini

```diff
@@ -31,8 +31,6 @@ commands = {posargs:py.test -v\
            {envsitepackagesdir}/virtualenvrunner {toxinidir}/tests}
 
 [pytest]
-addopts = --pep8
```

Comment:
If pep8 checks are not used any more you could remove `pytest-pep8` from dependency list.

Reply:
ok